README.md

```diff
@@ -3,4 +3,4 @@
 ## APIs
 
 - [Login](/login)
-- [User](/users)
+- [User](/user)
```

Comment:
Shouldn't the folder be called `users`, since that's what the API is called? #consistency